src/orb.yml

```
@@ -129,6 +129,15 @@ executors:
       Use example: `executor: cypress/base-14`.
     docker:
       - image: cypress/base:14
+      
+  # Node 16.13.2 tag images from Cypress
+  base-16-13-2:
+    description: |
+      Single Docker container with the latest Node v16 and Cypress dependencies
+      see https://github.com/cypress-io/cypress-docker-images/tree/master/base.
+      Use example: `executor: cypress/base-16-13-2`.
+    docker:
+      - image: cypress/base:16.13.2
```

```suggestion
  # Node 16.14.0 tag image from Cypress
  base-16-14-0:
    description: |
      Single Docker container with Node 16.14.0 and Cypress dependencies
      see https://github.com/cypress-io/cypress-docker-images/tree/master/base.
      Use example: `executor: cypress/base-16-14-0`.
    docker:
      - image: cypress/base:16.14.0
```

I'm going to suggest that we update the Node 16 executor in this PR to `16.14.0` to avoid [a potential issue](https://github.com/cypress-io/cypress-docker-images/pull/603) with `16.13.2`.